### PR TITLE
[confluence] [kodi-resource] [kodi-language-resource] add icons and skin level image

### PR DESCRIPTION
**Pull request by un1versal**
This adds confluence skin level folder to match newly added repo for languages addons and
add a GENERIC icon for the resource icon types.

Lets see if we can avoid the drama and provide any concise feedback in order to address any comments.

@Montellese @ronie 

thx

### CODE REVIEW

**Comment by Montellese** [comment]
Do we actually show the icons for the extension points anywhere? Either way this looks good for me.

**Comment by MartijnKaijser** [comment]
this doesn't look good to me. Now you done the icon that you should have done for the other PR.

**Comment by MartijnKaijser** [comment]
Let me write it in plain English:

   addons/resource.language.en_gb   -> "Language"  (the icon you did in the other PR)
   addons/kodi.resource   ->  "resource"   (the icon you included in this PR)

**Comment by Montellese** [comment]
Ah sorry I didn't check the file names.
The resource icon should be for `addons/kodi.resource/`. The english language addon (and all other new language addons) will need their own icons (maybe the flag of the country?). But the one you included for `addons/skin.confluence/media/DefaultAddonLanguage.png` should be good.

**Comment by MartijnKaijser** [comment]
i'd keep the icon for each language simple. have this design
screenshot
and just put "language"

**Comment by ksooo** [comment]
I'd prefer if we would find a design which is meaningful enough without language dependend text. But not being an artist at all, this can only be my 2 cents wrt maintainability (effort for new languages).

BTW: Do we have other icons containing language dependend text?

**Comment by ronie** [comment]
for what it's worth, i agree with ksooo. we should try to avoid using text on icons.

**Comment by ksooo** [comment]
And please do not use flags. Those might be easily source for (political) trouble. Think of China/Taiwan, the both Koreas, ...

**Comment by uNiversaI** [comment]
@MartijnKaijser yes thats whay I did that as it was a generic for all languages.

So you want the old icon back with speech bubble and just put language as the only text on it? or no text at all?

**Comment by MartijnKaijser** [comment]
just the speech bubble for the en_gb addon

**Comment by uNiversaI** [comment]
Right.

@Montellese the extension point I dont think its visible anywhere.

@MartijnKaijser done?! I hope.

I think part of the confusion was due to the tittle saying ***kodi-resource*** and not ***kodi-language-resource*** on other PR. as it was really for language part not for the extension point.

Now this PR adds all 3 icons and each is labelled correctly accordingly to what they are.

Should be ok now.

**Comment by MartijnKaijser** [comment]
I fou drop "kodi" from the following icon i'm fine :) thx
 addons/kodi.resource/icon.png

**Comment by Sorien** [comment]
i'd use icon of book like https://www.iconfinder.com/icons/339919/book_culture_document_documents_information_library_page_paper_sheet_text_icon#size=128 instead of gear

**Comment by uNiversaI** [comment]
Im open to suggestions, but wont use those icons you pointed to due to licensing nonsense and its not a editable vector.

I updated the icon to @MartijnKaijser request, and if he wants, I can submit this icon instead.
icon1

I have no preference, because this icon will probably remain unseen to anyone as extension points not visible.

**Comment by MartijnKaijser** [comment]
thx
